assets/prototype/domain/eventEditor/containers/queries/useInitQueries.js

```diff
@@ -0,0 +1,31 @@
+import get from 'lodash/get';
+import { useQuery } from '@apollo/react-hooks';
+import { GET_DATETIMES, GET_TICKETS } from './dates';
+
+const useInitQueries = ({ eventId }) => {
+	const { loading: loading1, data: datetimesData } = useQuery(GET_DATETIMES, {
+		variables: {
+			where: {
+				eventId
+			}
+		}
+	});
+	const datetimes = get(datetimesData, ['datetimes', 'nodes']);
+
+	const datetimeIn = datetimes && datetimes.map(({ id }) => id);
+	const { loading: loading2, data: ticketsData } = useQuery(GET_TICKETS, {
+		variables: {
+			where: {
+				datetimeIn
+			}
+		}
+	});
+	const ticketsNodes = get(ticketsData, ['tickets', 'edges']);
+	const tickets = ticketsNodes && ticketsNodes.map(({ node }) => node);
+
+	console.log({ loading1, loading2 });
```

Comment:
would be useful to rename these to `loadingDates` and `loadingTickets` and then return to the editor so that they can be passed along to the `datesList` and `ticketsList` components, since I don't believe the `readQuery` calls return a loading var at all, or do they?